CIP-Improved_Rewards_Scheme_Parameters/README.md

```diff
@@ -0,0 +1,239 @@
+---
+CIP: 82
+Title: Improved Rewards Scheme Parameters
+Status: Proposed
```

Comment:
```suggestion
Status: Inactive (no IOG support)
```
... or some such "justification" which indicates the problem.  @KtorZ we may need your help choosing a justification tag that will be (grimly) applied to all RSS proposals.  Mentioning the specific company name seems undiplomatic but it is not inaccurate or open to interpretation either so I believe it should be stated baldly as such.

Reply:
I'd simply go for: "project not enlisted for CIP process", which goes back to CIP-0001 and the need for a project to explicitly states their engagement to the CIP process.